openedx/core/djangoapps/cors_csrf/middleware.py

```diff
@@ -145,3 +128,120 @@ def process_response(self, request, response):
             )
 
         return response
+
+
+class CrossDomainCsrfViewMiddleware(django_csrf.CsrfViewMiddleware):
```

Comment:
This looks like a direct copy from the below. Without explaining why the copy was made. I think this would be problematic and won't be easily maintained.

Let's not move with this forward unless absolutely necessary.

https://github.com/django/django/blob/bc1f79d0a01d085500aa82ff29800403291a91a4/django/middleware/csrf.py#L137-L332

Reply:
Moved to https://github.com/Edraak/edraak-platform/pull/92